cppwamp/include/cppwamp/internal/endian.ipp

```diff
@@ -5,7 +5,7 @@
                     http://www.boost.org/LICENSE_1_0.txt)
 ------------------------------------------------------------------------------*/
 
-#include "endian.h"
+#include "endian.hpp"
```

Comment:
`#include "endian.hpp"` should actually be removed entirely. It's `endian.hpp` that includes `endian.ipp`, and not the other way around.